specification/components/schemas/headers/request/OboUserId.yaml

```diff
@@ -0,0 +1,8 @@
+in: header
+name: nhsd-ers-obo-user-id
+description: | 
+  The (SDS) user ID of the user that the authenticating user wishes to act on behalf of (OBO). Currently "on behalf of" is only supported for a Service Provider Clinician Admin (SPCA) acting on behalf of a Service Provider Clinician (SPC). Where an OBO User ID is supplied then the authenticating user should therefore be an SPCA and the OBO User ID should be the ID of an appropriate SPC.
```

Comment:
If it hasn't already been reviewed, it's very likely that NHSD will want to review and sign-off this wording. OdsCodeOnlyUserRestricted.yaml when through a similar review cycle